src/bundle/Controller/Content/ContentTreeController.php

```diff
@@ -116,7 +118,21 @@ public function loadSubtreeAction(Request $request): Root
         $sortOrder = $request->query->getAlpha('sortOrder', Query::SORT_ASC);
 
         $locationIdList = array_column($loadSubtreeRequest->nodes, 'locationId');
-        $locations = $this->locationService->loadLocationList($locationIdList);
+        $locations = [];
+
+        // Always load root location using `sudo`
+        if (in_array(self::ROOT_LOCATION_ID, $locationIdList, true)) {
+            $locations = $this->repository->sudo(function (): iterable {
+                return (array)$this->locationService->loadLocationList([self::ROOT_LOCATION_ID]);
```

Comment:
**NEVER** use `(array)` cast. It does not work how you expect it to.

https://www.php.net/manual/en/language.types.array.php#language.types.array.casting

You're dealing with `array|Traversable` (`array|object`) here.

> If an [object](https://www.php.net/manual/en/language.types.object.php) is converted to an [array](https://www.php.net/manual/en/language.types.array.php), the result is an [array](https://www.php.net/manual/en/language.types.array.php) whose elements are the [object](https://www.php.net/manual/en/language.types.object.php)'s properties. The keys are the member variable names, with a few notable exceptions: integer properties are unaccessible; private variables have the class name prepended to the variable name; protected variables have a '*' prepended to the variable name. These prepended values have NUL bytes on either side. Uninitialized [typed properties](https://www.php.net/manual/en/language.oop5.properties.php#language.oop5.properties.typed-properties) are silently discarded.